src/lib/dev_util/signature.ts

```diff
@@ -0,0 +1,38 @@
+const keccak256 = require("keccak256");
+import { sha256 } from "js-sha256";
+import { extendHexString } from "./utils";
+
+// Recovers the message hash of an arbitrary message signature
+// https://github.com/arx-research/libhalo/blob/master/docs/halo-command-set.md#command-sign
```

Comment:
There's also a mode where the card signs a raw digest; I wonder if that is worth using over the EIP-712 method so we don't need an additional keccak256 import. I want to keep the package.json as slim as possible for this website

Reply:
Sounds good, I've replaced this with a placeholder that uses sha256 - we should test this heavily when we get the cards